.github/actions/deploy-centos/action.yml

```diff
@@ -26,7 +27,9 @@ runs:
         if [[ "$CENTOS_VERSION" == "6" ]]; then
           sed -i 's/mirror\.centos\.org\/centos/vault.centos.org/g;s/6\/sclo/6.10\/sclo/g;s/mirrorlist/#mirrorlist/g;s/#baseurl/baseurl/g' /etc/yum.repos.d/CentOS-SCLo-scl-rh.repo
         fi
-        yum -y install $SCL_ENABLE rh-java-common-ant boost-devel ccache clang gcc-c++ gcc-gfortran java-1.8.0-openjdk-devel ant python python3-devel python3-pip swig file which wget unzip tar bzip2 gzip xz patch autoconf-archive automake make libtool bison flex perl-core nasm alsa-lib-devel freeglut-devel gtk2-devel libusb-devel libusb1-devel curl-devel expat-devel gettext-devel openssl-devel bzip2-devel zlib-devel SDL2-devel libva-devel libxkbcommon-devel libxkbcommon-x11-devel xcb-util* fontconfig-devel libffi-devel ragel ocl-icd-devel GeoIP-devel pcre-devel ssdeep-devel yajl-devel
+        curl -L "https://negativo17.org/repos/epel-multimedia.repo" -o /etc/yum.repos.d/epel-multimedia.repo
```

Comment:
Does that make YUM pick up libraries from that repo in priority? Is there a way to not make it do that? If not, since we're just interested in libva here, what about downloading the RPMs manually and installing only those?

Reply:
Then again, it doesn't look like there's any other package from there than libva that overlaps with EPEL anyway? What do you think?

Reply:
We should lower the priority, just to be safe, because we don't know if the owner adds any packages in the future.

Reply:
I'm now setting the priority to 99 (lowest) via sed

Reply:
I've reverted that last one since it's failing with linux-x86. It's fine like that.